src/Driver.php

```diff
@@ -54,7 +54,7 @@ class Driver
      *
      * @var array
      */
-    private $defaultOptions = [
+    private const DEFAULT_OPTIONS = [

```

Comment:
Class constant visibility is only allowed since PHP 7.1, the minimum PHP version supported is currently >=5.5.0. Could you fix it? Even better if you bump the required PHP version up to >=8.1, since we are working on the 3.0 version (https://github.com/gnello/php-mattermost-driver/issues/50#issuecomment-1540885924).

Reply:
Not sure about that. I'm currently running this changes from php7.3 onwards to 8.1. Looks like it is ok to keep them in 2.*. So I'd prefer to drop constant visibility.

Reply:
Ok then, in this case we have to at least require PHP 5.6 instead of 5.5. 
Anyway we can not merge these changes into the 2.*. version, as the the constructor signature of the `Driver` class is changing. 

My intention is to merge https://github.com/gnello/php-mattermost-driver/pull/52 and https://github.com/gnello/php-mattermost-driver/pull/54, require php8, search for other changes and tag the master with 3.0, how does it sound?

Reply:
So ok, lets require 8.0 as minimum